docs/contributing/styleguides/_category_.json

```diff
@@ -0,0 +1,5 @@
+{
+    "label": "Code Style Guides",
```

Comment:
Do we want to push this to production right now if it's coming soon?

Reply:
Just wanted something to link to that wasn't broken, the style guides should come in the next few days for Rust and Typescript

Reply:
Sounds good.